index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
Better have a readme file

Reply:
As I see in my VS code I have readme file. Should I add it in the HTML file or what?

Reply:
My mistake, you have it in codes. I just saw the file changes